code/datums/spells/ethereal_jaunt.dm

```diff
@@ -15,44 +15,77 @@
 
 	var/jaunt_duration = 50 //in deciseconds
 	var/jaunt_in_time = 5
+
+	///Do we do the glowy wizard thing on the tile the wizard entered / exit
+	var/has_jaunt_effect = TRUE
 	var/jaunt_in_type = /obj/effect/temp_visual/wizard
 	var/jaunt_out_type = /obj/effect/temp_visual/wizard/out
+	/// Do we show the fun blue water effect spread out thing
+	var/has_smoke_jaunt_effect = TRUE
+
+	/// Sound to play on jaunting
+	var/jaunt_enter_sound = 'sound/magic/ethereal_enter.ogg'
+
+	/// Sound to play when we start exiting jaunt
+	var/jaunt_exit_sound = 'sound/magic/ethereal_exit.ogg'
+
+	/// Do we unstun (currently shadowling)
+	var/unstuns = FALSE
+
+	/// Do we play a message when activating / deactivating the spell, in the format of "<span class='warning'>(user's name) (message to be displayed)</span>"
+	var/plays_message = FALSE
+	var/message_in = "Should not"
+	var/message_out = "See this"
```

Comment:
Since ya documented your other vars, could you do it for these two?

Reply:
Will do!